STL_Extension/doc/STL_Extension/examples.txt

```diff
@@ -1,4 +1,5 @@
 /*!
+\example STL_Extension/bisect_failures_example.cpp
```

Comment:
Please drop the suffix  `_example`. There are many without it in this directory.